src/com/lftechnology/java/training/alina/jdbc/dao/employee/impl/EmployeeDaoImpl.java

```diff
@@ -0,0 +1,162 @@
+package com.lftechnology.java.training.alina.jdbc.dao.employee.impl;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.lftechnology.java.training.alina.jdbc.constants.Constants;
+import com.lftechnology.java.training.alina.jdbc.dao.employee.EmployeeDao;
+import com.lftechnology.java.training.alina.jdbc.dbutils.DbFacade;
+import com.lftechnology.java.training.alina.jdbc.domain.Database;
+import com.lftechnology.java.training.alina.jdbc.domain.Employee;
+import com.lftechnology.java.training.alina.jdbc.service.DateTimeService;
+import com.lftechnology.java.training.alina.jdbc.service.UserService;
+
+public class EmployeeDaoImpl implements EmployeeDao {
+
+    private static final Logger LOGGER = Logger.getLogger(EmployeeDaoImpl.class.getName());
+
+    @Override
+    public List<Employee> findByPk(Integer userId) {
+        List<Employee> employeeList = new ArrayList<Employee>();
+        Map<Integer, Object> params = new HashMap<Integer, Object>();
+        params.put(1, userId);
+        params.put(2, Constants.NOT_TERMINATED);
+        params.put(3, Constants.NOT_DELETED);
+        Database database = UserService.setDatabaseParams(params);
+        String sql =
+                "Select * from user as u inner join employee as e on (u.user_id=e.user_id) where (u.user_id=? and u.is_terminated=? and e.is_deleted=?) ";
+        try {
+            Connection connection = DbFacade.getDbConnection();
+            PreparedStatement preparedStatement = connection.prepareStatement(sql);
+            preparedStatement = DbFacade.setParameterizedObjects(database, preparedStatement);
+            ResultSet result = preparedStatement.executeQuery();
+            while (result.next()) {
+                UserService userService = new UserService();
+                employeeList.add(userService.map(result));
+            }
+            preparedStatement.close();
+        } catch (SQLException sqe) {
+            LOGGER.log(Level.WARNING, "SQLException : {0}", new Object[] { sqe });
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "SQLException : {0}" 6 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)